modules/googleapis-cc/0.1.0/source.json

```diff
@@ -0,0 +1,7 @@
+{
+    "integrity": "sha256-hznHbmgfkAkjuQDJ3w73XPQh05yrtUZQxLmtGbanbYU=",
+    "url": "https://github.com/fmeum/bazel-central-registry/releases/download/v1.0.0/empty.zip",
```

Comment:
TODO: Replace with a better source after https://github.com/bazelbuild/bazel-central-registry/pull/3474.

Reply:
Couldn't we just allow url to be not set when overlay is used?

Reply:
We could, but that would require raising the minimum Bazel version to at least 8.1.0 (the earliest we can ship this).